### ISSUE TITLE
BatchNorm Layer save/load Issue?

### ISSUE THREAD

**Issue by piiswrong** [dmlc/mxnet#989]
I'm having some issue with saving/loading models. My model was snapshotted at 70% acc and when I load it, it starts from 50% and quickly climb to 70 percent within 30 batches. If I train from scratch the same increase from 50-70 takes several epochs. 

So it looks like something was corrupted during saving and was recovered quickly. I suspect it's due to BN layer. Any suggestions?
